build.sbt

```diff
@@ -60,12 +55,11 @@ val buildDebSettings = Seq(
 
 val `membership-common` =
   (project in file("membership-common"))
-    .enablePlugins(DynamoDBLocalPlugin, SbtPgp, Sonatype)
```

Comment:
we don't ship membership-common from this project, so I deleted all the related stuff